src/components/fields/FontSelector.js

```diff
@@ -2,29 +2,21 @@ import Dropdown from './Dropdown';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-/* eslint-disable react/prop-types */
-const styledRenderer = ({value, label}) => <span style={{fontFamily: value}}>{label}</span>;
-/* eslint-enable react/prop-types */
-
-const FontSelector = (props, context) => {
-  return (
-    <Dropdown
-      {...props}
-      options={context.fontOptions}
-      valueRenderer={styledRenderer}
-      optionRenderer={styledRenderer}
-    />
-  );
-};
+const FontSelector = (props, context) => (
+  <Dropdown
+    {...props}
+    options={context.fontOptions.map(({value, label}) => ({
+      label: <span style={{fontFamily: value}}>{label}</span>,
+      value,
+    }))}
+    clearable={true}
+  />
+);
 
 FontSelector.propTypes = {
   ...Dropdown.propTypes,
 };
 
-FontSelector.defaultProps = {
```

Comment:
shouldn't we keep the default value we had? and let clearable be an option we set through props..